sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/administration/models/EmailNotificationHook.java

```diff
@@ -118,12 +118,16 @@ public EmailNotificationHook setExternalLink(String externalLink) {
 
     /**
      * Sets the user e-mails and clientIds with administrative rights to manage the hook.
+     * <p>
+     * The administrators have total control over the hook, being allowed to update or delete the hook.
+     * Each element in this list represents a user with administrator access, but the value of each string element
+     * as either user email address or clientId uniquely identifying the user service principal.
      *
-     * @param emails The emails and clientIds
+     * @param admins The admins value
```

Comment:
```suggestion
     * @param admins A list containing email or clientId of admins
```
Make similar changes in other `@param` docs as well.